devices/pine64-pinephone/u-boot/default.nix

```diff
@@ -58,29 +58,13 @@ in
     # https://patchwork.ozlabs.org/patch/1202024
     (pw "1202024" "0c196zk1s3pq3wdv909sxmjgqpll2hwb817bpbghkfkyyknl96vg")
 
-    # sunxi: support asymmetric dual rank DRAM on A64/R40
-    # https://patchwork.ozlabs.org/patch/1312857
-    (pw "1312857" "0w8caf936y7f1r2lvdqar9n3dfkm461lm3k5hfwxd7biwyd386nk")
-
-    # sunxi: Add arm64 FEL support
-    # https://patchwork.ozlabs.org/patch/1402908
-    (pw "1402908" "1pm6zr7xgafw04rb95237q1mxb8xa6k2lmhzfsj0x0gxal96rw2v")
-
-    # sunxi: dram: h6: Improve DDR3 config detection
-    # https://patchwork.ozlabs.org/patch/1410521
-    (pw "1410521" "1vmalpf3wl3ii592qsh57g11qfrkmx40hlg8mza1330np2xfgb7v")
-
     # Enable led on boot to notify user of boot status
     # https://gitlab.com/pine64-org/u-boot/-/commit/1a72bd9f5b0d75361b0852ed515d40a47c1e9bfe
     (pine64Patch "1a72bd9f5b0d75361b0852ed515d40a47c1e9bfe" "0z6md01qcsf6xwb6hyn4cgxjz2d0dhv05k5hrqp9kwsh9yghchqq")
 
     # pinephone: Add volume_key environment variable
     # https://gitlab.com/pine64-org/u-boot/-/commit/2ce71f93c215bf87a9a64a93c3b2c28a86158e5b
-    (pine64Patch "2ce71f93c215bf87a9a64a93c3b2c28a86158e5b" "1x5p0p6zyd4y7yigyikhlwhrywqmd0kpkdnb38583svqzadbr9gw")
-
-    # sunxi: DT: A64: Add wifi to PinePhone DTS
-    # https://gitlab.com/pine64-org/u-boot/-/commit/b2f2ff6913788437dddf64fa9bcef875e260ec1e
-    (pine64Patch "b2f2ff6913788437dddf64fa9bcef875e260ec1e" "0rj3nx067r4y1n7s7l953v8y23l17vc0w5j47dqx3fim8frnq7j3")
+    ./volume-key-patch.patch
```

Comment:
This can be safely dropped as it's unused in the end.

If we were to want to use this, I have a better (bad still) UCLASS_BUTTON implementation cooking up.